iorgen/markdown.py

```diff
@@ -53,7 +53,7 @@
         'next lines': 'Sur les lignes suivantes,',
         'one line': 'Une ligne avec',
         'output': 'Sortie',
-        'str': 'une chaine de **{}** caractères',
+        'str': 'une chaîne de **{}** caractères',
```

Comment:
It is by choice that I wrote `chaine` that way. It is the recommended spelling since 1990.
And since we have a educational purpose, it all the more important to use the recommended spelling, like it is recommended in Éducation Nationale.